bottlecap/src/metrics/enhanced/lambda.rs

```diff
@@ -27,6 +27,19 @@ impl Lambda {
         self.increment_metric(constants::TIMEOUTS_METRIC)
     }
 
+    pub fn set_init_duration_metric(&self, init_duration_ms: f64) -> Result<(), errors::Insert> {
```

Comment:
Did you figure out the issue on why sometimes it wasn't appearing? And is this a remediation to it?

Reply:
Not exactly. We don't want to wait on the report log to set init duration because we'll know it from the init report. So this is just an improvement.